pages/index.vue

```diff
@@ -161,15 +165,15 @@
                 <img src="/img/volunteer.svg" alt="Volunteer" />
               </div>
             </div>
-            <h3 class="text-2xl font-extrabold text-center">Volunteer</h3>
+            <h3 class="text-2xl font-extrabold text-center">
+              {{ $t('features.volunteer') }}
+            </h3>
             <p class="text-center">
-              We build a community, where everyone enters with mindset “I can
-              help”, so whenever you need help you can count on it. We are one
-              big family worldwide. We help each other.<br />
+              {{ $t('features.volunteerText') }}
               <router-link
                 to="/get-involved"
                 class="underline font-bold hover:no-underline"
-                >Get Involved</router-link
+                >{{ $t('features.getInvolved') }}</router-link
```

Comment:
Every feature translation I would structure like this:

```
features.volunteer.title
features.volunteer.description
features.volunteer.action
```

Reply:
You can read more about it here - https://github.com/we-dance/platform/discussions/70